include/NovelRT/Maths/GeoMatrix4x4F.h

```diff
@@ -165,11 +755,36 @@ namespace NovelRT::Maths
             return GeoMatrix4x4F(glm::ortho(left, right, bottom, top, zNear, zFar));
         }
 
+        /**
+         * @brief Creates a matrix with the given scale components.
+         *
+         * @details
+         * The scale components are placed in the decending diagonal of the matrix: \f[
+         *      \begin{bmatrix}
+         *      x & 0 & 0 & 0\\
+         *      0 & y & 0 & 0\\
+         *      0 & 0 & z & 0\\
+         *      0 & 0 & 0 & 1
+         *      \end{bmatrix}
+         * \f]
+         * @param x The scale component over the X axis.
+         * @param y The scale component over the Y axis.
+         * @param z The scale component over the Z axis.
+         * @returns A matrix with scale components applied.
+         */
         static GeoMatrix4x4F CreateFromScale(float x, float y, float z) noexcept
         {
             return GeoMatrix4x4F(glm::scale(glm::vec3(x, y, z)));
         }
 
+        /**
+         * @brief Creates a matrix for observing a point in space from a given location.
+         *
+         * @param eye The position of the observer.
+         * @param centre The position the observer should look towards.
+         * @param up The updirection for the observer.
+         * @returns
```

Comment:
appears unfinished?

Reply:
Documentation for the return has been added